src/command_modules/azure-cli-network/azure/cli/command_modules/network/_validators.py

```diff
@@ -308,6 +315,10 @@ def process_public_ip_create_namespace(namespace):
     if namespace.dns_name:
         namespace.public_ip_address_type = 'dns'
 
+def load_cert_file(namespace):
+    if os.path.isfile(namespace.public_cert_data):
```

Comment:
> if os.path.isfile(namespace.public_cert_data): [](start = 3, length = 47)

Why this check? If the specified path isn't a file, why fail silently? Why not let it throw?